stratum/hal/lib/phal/BUILD

```diff
@@ -91,7 +91,7 @@ phal_defines = select({
 }) + select({
     ":with_tai": tai_defines,
     ":without_tai": [],
-    "//conditions:default": [],
+    "//conditions:default": tai_defines,
```

Comment:
Default will be no TAI. Most platforms don't have optical ports.

Reply:
That was my attempt to fix the build on CI. Though, it's fixed now through correct target deps, so removed this. Please review.